aws/v20.0.0/README.md

```diff
@@ -8,7 +8,7 @@ This release will also be used as a base for `Giant Swarm Vintage` to Giant Swar
 
 > **WARNING:** After upgrading to `19.3.0`, it is highly advised to begin removal of all PSPs from the cluster. Kubernetes `v1.25` removes the Pod Security Policy resource from the API, meaning workloads (like Helm charts) which still contain PSPs will fail to install after the upcoming Giant Swarm `v20` release.
 
-> **WARNING:** `Observability-bundle` will be upgraded to `v1.1.0` and it contains breaking changes by simplify configuration for the bundled apps. Please check our [upgrade guide](https://github.com/giantswarm/observability-bundle/blob/main/docs/upgrade.md) or reach out to your Account Engineer for more details.
+> **WARNING:** `observability-bundle` will be upgraded to `v1.1.0`, which contains breaking changes to the configuration for the bundled apps. Please check our [upgrade guide](https://github.com/giantswarm/observability-bundle/blob/main/docs/upgrade.md) or reach out to your Account Engineer for more details.
```

Comment:
```suggestion
> **WARNING:** `observability-bundle` will be upgraded to `v1.1.0`, which contains breaking changes to the configuration for the bundled apps to simplify configuration, especially for GitOps. Please check our [upgrade guide](https://github.com/giantswarm/observability-bundle/blob/main/docs/upgrade.md) or reach out to your Account Engineer for more details.
```

Reply:
@T-Kukawka do you think we could bump the observability bundle to 1.2.4?

Reply:
if we are fine that it does not break anything, maybe worth to test real quick? then yes.

Reply:
It should not but can we do a test release quick? :D

Reply:
@QuentinBisson u can just take that release.yaml, apply it on the Test MC or on `garfish` named e.g. 20.0.1, adjust your app in the yaml and deploy the cluster with that release basically - we do not cut test releases

Reply:
sure I'm aware. I'll test tomorrow because I'm supposed to be off today :D

Reply:
sure 👍 no worries

Reply:
Marie ran a test and it worked like a charm with version 1.3.0 even :D

Reply:
https://github.com/giantswarm/releases/pull/1217

Reply:
approved